### PR TITLE
Apr SL release notes

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr sets to add all the SL and MF updates for apr via a release note.  It also updates the tableau page saying we're going deprecate the metrics and dimension data source soon.

[Notion draft](https://www.notion.so/dbtlabs/Semantic-Layer-April-Release-Notes-26eaa08576ed47979bb54788d026ca3d)

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/70-Apr-2024/apr-sl.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- **Display metric label** &mdash; Metrics are now displayed by their labels as `metric_name`.
+
+### Updates
+
+- **Metadata in metrics** &mdash; [Metrics](/docs/build/metrics-overview) now supports the [`meta` option](/reference/resource-configs/meta) under the [config](/reference/resource-properties/config) property. Previously, we only supported the now deprecated `meta` tag.
+
+**Google Sheets**
+
+- **Explore saved queries** &mdash; Added [support](/docs/use-dbt-semantic-layer/gsheets#using-saved-queries) to allow jumping off from or exploring MetricFlow-defined saved queries directly.
+- **Query dimensions** &mdash; Added support to query dimensions without metrics. Previously, you needed a dimension.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/70-Apr-2024/apr-sl.md:29]: waiting on @DevonFulcher to confirm if this made it

File: website/docs/docs/dbt-versions/release-notes/70-Apr-2024/apr-sl.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+sidebar_label: "Updates and fixes: dbt Semantic Layer"
+sidebar_position: 06
+tags: [Apr-2024]
+date: 2024-04-30
+---
+
+The dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability for the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and [MetricFlow](/docs/build/about-metricflow). This release note includes fixes, updates, and deprecations for in April 2024.
+
+:::info Updated release note layout
+We've updated the layout of our release notes to provide a single view of the changes. Check out the [updated layout](/docs/dbt-versions/2024-release-notes) and let us know what you think by [opening up a GitHub issue](https://github.com/dbt-labs/docs.getdbt.com/issues)!
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/70-Apr-2024/apr-sl.md]:
```suggestion
We've updated the layout of our release notes to provide a single view of the changes. Check out the [updated layout](/docs/dbt-versions/2024-release-notes) and let us know of any problems by [opening up a GitHub issue](https://github.com/dbt-labs/docs.getdbt.com/issues)!
```

File: website/docs/docs/dbt-versions/2024-release-notes.md [hunk 3]
```diff
@@ -21,6 +21,17 @@ dbt Cloud release notes for 2024.
 
   </expandable>
 
+**dbt Semantic Layer**
+- **New**: Use Saved selections to [save your query selections](/docs/use-dbt-semantic-layer/gsheets#using-saved-selections) within the [Google Sheets application](/docs/use-dbt-semantic-layer/gsheets). They can be made private or public and refresh upon loading.
+- **New**: Metrics are now displayed by their labels as `metric_name`.
+- **Enhancement**: [Metrics](/docs/build/metrics-overview) now supports the [`meta` option](/reference/resource-configs/meta) under the [config](/reference/resource-properties/config) property. Previously, we only supported the now deprecated `meta` tag.
+- **Enhancement**: In the Google Sheets application, we added [support](/docs/use-dbt-semantic-layer/gsheets#using-saved-queries) to allow jumping off from or exploring MetricFlow-defined saved queries directly.
+- **Enhancement**: In the Google Sheets application, we added support to query dimensions without metrics. Previously, you needed a dimension.
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/2024-release-notes.md:29]: Waiting for confirmation for this ^

**Review by runleonarun — APPROVED** [review]
Looks good.

File: website/docs/docs/dbt-versions/release-notes/70-Apr-2024/apr-sl.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-versions/release-notes/70-Apr-2024/apr-sl.md:29]: Confirmed it made it!

File: website/docs/docs/dbt-versions/2024-release-notes.md (hunk 3 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-versions/2024-release-notes.md:29]: Confirmed it made it! Ty!

**Comment by mirnawong1** [comment]
Thank you so much @runleonarun - merging this now!

File: website/docs/docs/dbt-versions/release-notes/70-Apr-2024/apr-sl.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
+title: "dbt Semantic Layer updates and fixes for April 2024"
+description: "Apr 2024: Enhanced Google Sheets integration include new features for metric labels, saved selections, and time filters. Tableau integration improvements include data source update and bug fixes."
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/70-Apr-2024/apr-sl.md]:
```suggestion
description: "Apr 2024: Enhanced Google Sheets integration include new features for metric labels, saved selections, and time filters. Tableau integration improvements include data source updates."
```

**Pull request merged**